src/runtime/emit-code.ts

```diff
@@ -84,7 +84,7 @@ export const generateCodeToEmit = ({ code, args }: { code: string; args: StoryOb
     // replace single arg references with their actual value,
     // eg. myProp={args.something} => myProp={"actual"}
     // or <h1>{args.something}</h1> => <h1>"actual"</h1>
-    .replace(/args(?:[\w\d_$\.\?\[\]"'])+/g, (argPath) => {
+    .replace(/args(?:[\w\d_$.?[\]"'])+/g, (argPath) => {
```

Comment:
are you sure this is correct?

Reply:
I did a quick test: https://regex101.com/r/zmPmKz/1
There was a rule complaint: https://eslint.org/docs/latest/rules/no-useless-escape#rule-details

Unless there are edge cases, I can't be 100% sure it is correct 😅. We have tests which are passing, and I have confidence in ESLint maintainers - their [test for this rule](https://github.com/eslint/eslint/blob/40dd2998cedddb75e0514b2c5cc855293c85da41/tests/lib/rules/no-useless-escape.js#L12) is quite extensive.

Reply:
SGTM